apps/src/code-studio/components/SoundList.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import SoundListEntry from './SoundListEntry';
+import Sounds from '../../Sounds';
+import {searchAssets} from '../assets/searchAssets';
+var soundLibrary = require('../soundLibrary.json');
+
+/**
+ * A component for managing sounds from soundLibrary.json.
+ */
+var SoundList = React.createClass({
+  propTypes: {
+    assetChosen: React.PropTypes.func.isRequired,
+    search: React.PropTypes.string.isRequired,
+    category: React.PropTypes.string.isRequired,
+    selectedSound: React.PropTypes.object.isRequired
+  },
+
+  componentWillMount: function () {
```

Comment:
nit: throughout this file and elsewhere, I think you can just do:
`componentWillMount() {`
instead of
`componentWillMount: function () {`